core/tasks.py

```
@@ -1,13 +1,26 @@
 import json
+from datetime import datetime, timedelta
```

Ruff (0.11.9)

`datetime.datetime` imported but unused

Remove unused import: `datetime.datetime`

(F401)